app/controllers/registers_controller.rb

```diff
@@ -23,6 +23,7 @@ def in_progress
   def show
     @register = Register.find_by_slug!(params[:id])
     @records = recover_records(@register.fields_array, params)
+    @is_empty_register = @register.records.where(entry_type: 'user').none?
```

Comment:
we can move this into the model `/app/models/register.rb`

e.g.
```
  def empty_register?
    records.where(entry_type: 'user').none? 
  end
```
then in the template `@register.empty_register?`